cli/cmd/package_manifest.go

```diff
@@ -236,3 +334,16 @@ func (c *cliState) checkPackageManagerWithNativeCommand(manager string) bool {
 	waitStatus := cmd.ProcessState.Sys().(syscall.WaitStatus)
 	return waitStatus.ExitStatus() == 0
 }
+
+func removeEpochFromPkgVersion(pkgVer string) string {
+	if !strings.Contains(pkgVer, ":") {
+		return pkgVer
+	}
+
+	pkgVerSplit := strings.Split(pkgVer, ":")
+	if len(pkgVerSplit) == 2 {
+		return pkgVerSplit[1]
+	}
+
+	return pkgVer
+}
```

Comment:
Can reduce this a little?
```suggestion

func removeEpochFromPkgVersion(pkgVer string) string {
	if strings.Contains(pkgVer, ":") {
	  pkgVerSplit := strings.Split(pkgVer, ":")
	  if len(pkgVerSplit) == 2 {
		  return pkgVerSplit[1]
	  }
	}

	return pkgVer
}
```

Reply:
Sounds good! I'll change it 💯